src/core/container.jl

```diff
@@ -43,7 +43,7 @@ function AdvancedPS.reset_model(f::TracedModel)
 end
 
 function AdvancedPS.reset_logprob!(f::TracedModel)
-    DynamicPPL.resetlogp!(f.varinfo)
+    DynamicPPL.resetlogp!!(f.varinfo)
```

Comment:
This also relies on the fact that it mutates `f.varinfo`.

Reply:
Yup, but `TracedModel` isn't compatible with immutable `AbstractVarInfo` :confused: